ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml

```diff
@@ -87,8 +87,38 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: perfscale-rosa-multiaz-data-path-9nodes-periodic
```

Comment:
should we put this in a separate pr since we are focusing on renaming

Reply:
Looks like that got in there accidentally, I removed it.